src/content/docs/en/pages/devtools/cli/azion-cli-alpha/cache-settings/cache-settings.mdx

```diff
@@ -0,0 +1,302 @@
+---
+title: Cache Settings
+description: Create, list, describe, update, and delete your cache settings on the Azion platform through Azion CLI.
+meta_tags: cli, cache, devops
+namespace: documentation_cli_cache_settings
+menu_namespace: cliMenuAlpha
+permalink: /documentation/products/cli/cache-settings-crud/
+---
+
+## Create
+
+### Usage
+
+```bash
+$ azion create cache-setting --application-id 1673635839 --name "phototypesetting"
+```
+
+### Required flags
+
+#### application-id
+
+The `--application-id` flag sets the unique identifier for the edge application.
+
+#### name
+
+The `--name` flag sets the name of the Cache Settings configuration being created.
+
+### Optional flags
+
+#### adaptive-delivery-action
+
+The `--adaptive-delivery-action` option informs the Cache Settings configuration adaptive delivery action. The default value is `ignore`.
+
+#### browser-cache-settings
+
+The `--browser-cache-settings` option configures the amount of time that the content is cached in the web browser. The default value is `honor`.
+
+#### browser-cache-settings-maximum-ttl
+
+The `--browser-cache-settings-maximum-ttl` option informs Browser Cache Settings configuration maximum TTL.
+
+#### cache-by-cookies
+
+The `--cache-by-cookies` option defines if cache by cookies is active. The default value is `ignore`.
+
+#### cache-by-query-string
+
+The `--cache-by-query-string` option defines how the content should be cached according to variations of the Query String in URLs. The default value is `ignore`.
+
+#### cdn-cache-settings
+
+The `--cdn-cache-settings` option configures the amount of time Azion's Edge Applications take to cache the content. It can either Honor Origin Cache Headers or Override Cache Settings. The default value is `honor`.
```

Comment:
```suggestion
The `--cdn-cache-settings` option configures the amount of time Azion's edge applications take to cache the content. It can either Honor Origin Cache Headers or Override Cache Settings. The default value is `honor`.
```